Vagrantfile

```diff
@@ -52,16 +52,20 @@ def vm_cpus
 end
 
 Vagrant.configure("2") do |config|
-  # always use Vagrants insecure key
+  # always use Vagrant's insecure key
```

Comment:
This commit also breaks upstream compatibility in much bigger ways... :innocent: